src/rust/engine/src/lib.rs

```diff
@@ -232,7 +232,7 @@ pub extern "C" fn scheduler_create(
     path_globs: type_path_globs,
     directory_digest: type_directory_digest,
     snapshot: type_snapshot,
-    merged_directories: type_merge_directories_request,
+    directories_to_merge: type_merge_directories_request,
```

Comment:
I can't get Rust to compile. Fails with these errors:

```rust
error[E0560]: struct `types::Types` has no field named `directories_to_merge`
   --> src/lib.rs:235:5
    |
235 |     directories_to_merge: type_merge_directories_request,
    |     ^^^^^^^^^^^^^^^^^^^^ `types::Types` does not have this field
    |
    = note: available fields are: `construct_directory_digest`, `construct_snapshot`, `construct_file_content`, `construct_files_content`, `construct_process_result` ... and 16 others

error[E0609]: no field `directories_to_merge` on type `types::Types`
   --> src/nodes.rs:188:44
    |
188 |             && input == context.core.types.directories_to_merge =>
    |                                            ^^^^^^^^^^^^^^^^^^^^ unknown field
    |
    = note: available fields are: `construct_directory_digest`, `construct_snapshot`, `construct_file_content`, `construct_files_content`, `construct_process_result` ... and 16 others

error[E0609]: no field `directories_to_merge` on type `types::Types`
   --> src/nodes.rs:191:62
    |
191 |             self.select_product(&context, context.core.types.directories_to_merge, "intrinsic");
    |                                                              ^^^^^^^^^^^^^^^^^^^^ unknown field
    |
    = note: available fields are: `construct_directory_digest`, `construct_snapshot`, `construct_file_content`, `construct_files_content`, `construct_process_result` ... and 16 others

error[E0609]: no field `directories_to_merge` on type `&types::Types`
  --> src/tasks.rs:86:22
   |
86 |         input: types.directories_to_merge,
   |                      ^^^^^^^^^^^^^^^^^^^^

error: aborting due to 4 previous errors
```

Any ideas what I'm doing wrong? I know I need to change something in `lib.rs`.

Reply:
> error[E0609]: no field `directories_to_merge` on type `types::Types`

`types::Types` is this thing: https://github.com/pantsbuild/pants/blob/master/src/rust/engine/src/types.rs

Reply:
Aw, that's it. Thank you Stu!